test/lib/viewers/RuleViewer.test.ts

```diff
@@ -13,7 +13,7 @@ describe('RuleDetailViewer', () => {
 		expect(displayEvents).toHaveLength(1);
 		expect(displayEvents).toEqual([{
 			type: DisplayEventType.INFO,
-			data: 'Found 0 rules.'
+			data: '*DRAFT*: Found 0 rules.'
```

Comment:
I go back and forth on this... what are your thoughts about having the tests use the message catalog as well? There are pros/cons to both approaches (test maintainability when output changes vs test readability to confirm message are constructed properly).

Reply:
I waffle on it too. Typically, my preference is for the tests to be readable, which is maximized when you assert against the actual strings. But I'm willing to be flexible I think (probably would want to do it in a separate PR though...)